.eslintrc.json

```diff
@@ -40,6 +40,7 @@
   ],
   "rules": {
     // Disabled default rules
+    "no-debugger": "off",
```

Comment:
I'm open to what others think, but I personally like having it on so we don't end up with debugger statements in our committed code.  Maybe add `"no-debugger": "on",` so it can be more easily set to off for a while when we need to debug?